uofm_maintenance.module

```diff
@@ -92,6 +98,31 @@ function uofm_maintenance_regen_jp2_worker($object) {
 }
 
 /**
+ * Regenerate FITS derivative from OBJ.
+ *
+ * @param string|AbstractObject $object
+ *   the string PID or a Fedora Object
+ */
+function uofm_maintenance_regen_fits_worker($object) {
+  $obj = FALSE;
+  if (is_string($object)) {
+    module_load_include('inc', 'islandora', 'includes/utilities');
+    if (islandora_is_valid_pid($object)) {
+      $obj = islandora_object_load($object);
+    }
+  }
+  if (is_a($object, 'AbstractObject')) {
+    $obj = $object;
+  }
+  if (isset($obj['OBJ'])) {
+    module_load_include('inc', 'islandora_fits', 'includes/derivatives');
+    drush_log("Regenerate FITS for " . $obj->id, "ok");
+    islandora_fits_create_techmd($obj, TRUE, array('source_dsid' => 'OBJ'));
```

Comment:
So this is entirely up to you, but you could capture the output of `islandora_fits_create_techmd()` and use that to report success or failure. Some examples of the returned `array()` are:
Success:
https://github.com/Islandora/islandora_fits/blob/7.x/includes/derivatives.inc#L47-L56
Failures: 
https://github.com/Islandora/islandora_fits/blob/7.x/includes/derivatives.inc#L27-L37
https://github.com/Islandora/islandora_fits/blob/7.x/includes/derivatives.inc#L47-L56
https://github.com/Islandora/islandora_fits/blob/7.x/includes/derivatives.inc#L182-L194

Your choice, I should be doing that too. But I think I'll split this stuff out into sub-modules first.

Reply:
@ruebot I forgot about this PR, I have split out a bunch of stuff. So if you want to move your stuff into a sub-module that would be cool. Otherwise I might I just pull your stuff in.